### DISCUSSION TITLE
[HELP] Merge pull request without review [HELP]

### DISCUSSION THREAD

**Discussion by ghost** [community/community#30455] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi how can I merge pull request without review !! [HELP]

**Comment by radar-blip** [comment]
Here you go:  #30451

**Reply by ghost** [reply]
thanks

**Comment by MahaSegni** [comment, marked as answer]
I asked the same thing. the answer is that I had to get someone else to review the pull request

**Reply by ghost** [reply]
okay, thanks.
